lang/ar.js

```diff
@@ -2,44 +2,46 @@
 
 export default {
 	"components.alert.close": "إغلاق التنبيه",
-	"components.breadcrumbs.breadcrumb": "فتات الخبز",
+	"components.breadcrumbs.breadcrumb": "شريط التنقل",
 	"components.calendar.notSelected": "لم يتم التحديد.",
 	"components.calendar.selected": "تم التحديد.",
 	"components.calendar.show": "إظهار {month}",
 	"components.dialog.close": "إغلاق",
-	"components.form-element.defaultError": "{label} غير صالح.",
+	"components.form-element.defaultError": "{label} غير صالحة.",
 	"components.form-element.defaultFieldLabel": "الحقل",
-	"components.form-element.input.email.typeMismatch": "",
-	"components.form-element.input.number.rangeOverflow": "",
-	"components.form-element.input.number.rangeUnderflow": "",
-	"components.form-element.input.url.typeMismatch": "",
-	"components.form-element.valueMissing": "{label} مطلوب.",
-	"components.form-error-summary.errorSummary": "",
+	"components.form-element.input.email.typeMismatch": "{label} is not a valid email",
```

Comment:
These are english but I think we don't really need the translations yet so I can just remove the english?

Reply:
Do we typically keep English terms in these as a "fallback" of sorts?

Reply:
They're added in the `prepublishOnly` script when published to npm (https://github.com/BrightspaceUI/core/blob/master/package.json#L19) so we typically leave them empty in source as a way to see what's missing rather than fill them with english.